README.md

```diff
@@ -62,6 +63,7 @@ Workflow labels define the position of an issue or PR in out process:
 
 **Needs action labels**  
 Color: gold (#FFD700)  
+- `description needed`: This issue is not described well enough. Typically, it would be reassigned to the person who created it, so they can describe it better. This is an important step to make the issues the **_single source of truth_**.
```

Comment:
The idea was to use before we even think about requirements, to avoid having issues with no description that just sit there.